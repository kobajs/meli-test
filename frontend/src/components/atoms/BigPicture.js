import styled from "styled-components";

export default styled.img`
  height: 680px;
  width: 680px;
`;
