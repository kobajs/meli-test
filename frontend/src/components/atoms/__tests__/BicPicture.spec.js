import React from 'react';
import renderer from 'react-test-renderer';

import BigPicture from '../BigPicture';

it('renders correctly', () => {
  const tree = renderer
    .create(<BigPicture src="/static/logo.png" alt="logo" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});