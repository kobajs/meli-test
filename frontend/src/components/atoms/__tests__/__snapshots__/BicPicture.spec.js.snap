// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  alt="logo"
  className="BigPicture-sc-1ow2bxe-0 hhGULa"
  src="/static/logo.png"
/>
`;
